Formula/c/couchdb.rb

```diff
@@ -46,24 +37,16 @@ class Couchdb < Formula
   end
 
   def install
-    spidermonkey = Formula["spidermonkey@91"]
-    inreplace "configure", '[ ! -d "/usr/local/include/${SM_HEADERS}" ]',
-                           "[ ! -d \"#{spidermonkey.opt_include}/${SM_HEADERS}\" ]"
-    inreplace "src/couch/rebar.config.script" do |s|
-      s.gsub! "-I/usr/local/include/mozjs", "-I#{spidermonkey.opt_include}/mozjs"
-      s.gsub! "-L/usr/local/lib", "-L#{spidermonkey.opt_lib} -L#{HOMEBREW_PREFIX}/lib"
-    end
-
-    system "./configure", "--spidermonkey-version", spidermonkey.version.major.to_s
+    system "./configure", "--disable-spidermonkey", "--js-engine=quickjs"
     system "make", "release"
     # setting new database dir
     inreplace "rel/couchdb/etc/default.ini", "./data", "#{var}/couchdb/data"
     # remove windows startup script
-    rm_r("rel/couchdb/bin/couchdb.cmd")
+    rm("rel/couchdb/bin/couchdb.cmd")
     # install files
     prefix.install Dir["rel/couchdb/*"]
-    if File.exist?(prefix/"Library/LaunchDaemons/org.apache.couchdb.plist")
-      (prefix/"Library/LaunchDaemons/org.apache.couchdb.plist").delete
+    if (prefix/"Library/LaunchDaemons/org.apache.couchdb.plist").exist?
```

Comment:
Doesn't this mean it'll always exist or not exit for a given version of `couchdb`?

Reply:
I only modified style but haven't checked usage. I am guessing either it is only created on macOS or the removal logic is outdated.

Reply:
May have been removed in v2: https://github.com/apache/couchdb/commit/a19f7f70bf83bb2403f990668a5827c6775d617c

Will see if possible to just get rid of this.